### DISCUSSION TITLE
Limiting Integration Logs (Network Dataset) to HOME_NET

### DISCUSSION THREAD

**Discussion by 8bitjoe** [Security-Onion-Solutions/securityonion#11515] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

64

### Storage for /

200

### Storage for /nsm

2tb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Greetings, 

First of all i want to congratulate the Security Onion team for pushing out the 2.4.20 Hotfix that includes the elastic integrations many of us needed, (For me personally it was the Sonicwall integration).
However i do notice that these logs are visible in the dashboard and hunt interfaces, this raised a couple questions for me:

1) How can i restrict the visibility of these logs to the HOME_NET networks specified, this works for Suricata and zeek but in those dashboards i see logs from subnets i'm not actively monitoring in S.O, i have no way of filtering this in the Sonicwall Firewall.
2) Would a more sensible fix be to remove this information from the SOC interfaces and only query this data from kibana? is this possible? how do you determine which elastic data is visualized in SOC?
3) How can i create alerts for this logs so that they show in the alerts tab? Are there any good and recent resources on setting up playbook for S.O 2.4 so we can generate HOST alerts and alerts from integrations data?

Thank you very much for the effort and support.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> How can i restrict the visibility of these logs to the HOME_NET networks specified, this works for Suricata and zeek but in those dashboards i see logs from subnets i'm not actively monitoring in S.O, i have no way of filtering this in the Sonicwall Firewall.

One option would be to write your own custom queries that only show you the firewall logs that you want to see.

From https://docs.securityonion.net/en/2.4/dashboards.html#query-bar:
_If you would like to save your own personal queries, you can bookmark them in your browser. If you would like to customize the default queries for all users, please see the [SOC Customization](https://docs.securityonion.net/en/2.4/soc-customization.html#soc-customization) section._

> Would a more sensible fix be to remove this information from the SOC interfaces and only query this data from kibana? is this possible? how do you determine which elastic data is visualized in SOC?

Security Onion Console is designed to query Elasticsearch just like Kibana so that you don't necessarily need to switch to Kibana to run searches.

> How can i create alerts for this logs so that they show in the alerts tab? Are there any good and recent resources on setting up playbook for S.O 2.4 so we can generate HOST alerts and alerts from integrations data?

Have you reviewed the Playbook documentation?
https://docs.securityonion.net/en/2.4/playbook.html
